test_requirements.txt

```diff
@@ -0,0 +1,4 @@
+flake8==3.2.1
```

Comment:
Need to move `flake8` out of the main `requirements.txt`

Don't need `structlog` here as it's a dependency already covered in `requirements.txt`

Also need to add the running of `test_requirements.txt` to the `Makefile`